### ISSUE TITLE
"Overview" section placed at the end of document

### ISSUE THREAD

**Issue by ronaldtse** [iho-ohi/S-102-Product-Specification#108]
From @hasel001 :

The largest remaining issue involves (for the PDF) the Overview Section being placed much farther along in the document than where it belongs. See a recent output from a build a few minutes ago as an example.

Do you have any insight as to a solution to get the Overview section to go back to the beginning of the document?

Thanks again. We really appreciate your help.

Sample:
* [draftIncluding_03_and_09.pdf](https://github.com/user-attachments/files/15895662/draftIncluding_03_and_09.pdf)

**Comment by ronaldtse** [comment]
@hasel001 @RohdeBSH I have posted the issue here, please track it:
* https://github.com/metanorma/metanorma-iho/issues/227

**Comment by hasel001** [comment]
So, according to the screenshots and the results from our recently built artifacts, the numbering is now correct, in that the first chapter appearing in the document is numbered 1, etc.

However, in document.adoc, the order is as follows:

00-document-history.adoc
01-overview.adoc
02-spec-scope.adoc
...
12-metadata.adoc
aa-data-class-enc.adoc

But the order in the rendered document is:

00 Document History
01 Spec Scope
...
11 Metadata
12 Overview
AA Data Class Enc

In other words, overview is still at the end (despite its position in document.adoc).

@ronaldtse @opoudjis @Intelligent2013
